tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -27,21 +44,80 @@ public void run() {
     @Override
     public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
+             final var outputStream = connection.getOutputStream();
+             final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8))
+        ) {
+            final HttpRequest request = HttpRequest.from(bufferedReader);
 
-            final var responseBody = "Hello world!";
-
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            final HttpResponse response = handleRequest(request);
 
             outputStream.write(response.getBytes());
             outputStream.flush();
-        } catch (IOException | UncheckedServletException e) {
+        } catch (IOException | UncheckedServletException | URISyntaxException e) {
             log.error(e.getMessage(), e);
         }
     }
+
+    public HttpResponse handleRequest(final HttpRequest request) throws URISyntaxException {
+
+        final String uriPath = request.getPath();
+
+        if (uriPath.equals("/login")) {
+            final String path = LOGIN_PAGE;
+            if (request.getMethod().equals("GET")) {
+                return HttpResponse.of(HttpStatus.OK, path);
+            }
```

Comment:
path 변수를 사용하는 곳이 HttpResponse.of 밖에 없어서 다음과 같이 변경해보는건 어떨까요? 아래도 마찬가지입니다!
```suggestion
            if (request.getMethod().equals("GET")) {
                return HttpResponse.of(HttpStatus.OK, LOGIN_PAGE);
            }
```